ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -245,7 +245,12 @@ let set_diagnostics rpc doc =
                                  DiagnosticRelatedInformation.create ~location
                                    ~message)) )
                     in
-                    create_diagnostic ?relatedInformation ~range ~message
+                    let tags =
```

Comment:
can we keep the tags logic in one place? It will be a bit messy because the check is slightly different for dune and merlin (but we'll fix this soon enough)